### PR TITLE
Adds Travis and testing using Mocha

### LINKED ISSUE #7: Travis
Is there for a CI to do JavaScript? Once we have tests up we can get a CI to run those, but is there any point of having a CI for JavaScript without tests?

### PR DESCRIPTION
**Pull request by CalebJohn**
The mocha tests aren't functional yet, but it does use JsHint to check our code.
edit:
fixes #7

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+sudo: required
+dist: trusty
+node_js:
+  - "stable"
+install:
+  - npm install jshint
+  - npm install mocha
+script:
+  - jshint src/*.js
```
Comment (ExtraGravity) [inline .travis.yml]: This doesn't run jshint for sub dirs like states does it?

Comment (CalebJohn) [inline .travis.yml]: Nice catch! I'll fix it.

**Comment by clayjohn** [comment]
Are we good to merge this?

**Comment by ExtraGravity** [comment]
Yup looks good to me.

**Pull request merged**